src/ahbicht/validation/validation_results.py

```diff
@@ -168,3 +168,70 @@ def deserialize(self, data, **kwargs) -> ValidationResultInContext:
         so that the discriminator is the key and the validation_result the value.
         """
         return ValidationResultInContext(**data)
+
+
+@attrs.define(auto_attribs=True, kw_only=True)
+class ListOfValidationResultInContext:
+    """
+    Class to set validation result in context, for example with its discriminator.
+    """
+
+    validation_results: List[ValidationResultInContext] = attrs.field(
+        validator=attrs.validators.deep_iterable(
+            member_validator=attrs.validators.instance_of(ValidationResultInContext),
+            iterable_validator=attrs.validators.instance_of(list),
+        )
+    )
+
+    def replace_edi_seed_paths_with_bo4e_paths(self, edi_seed_to_bo4e_mappings: Dict[str, str]) -> None:
+        """
+        replaces the discriminators in the list of validation results with the respective values from the edi_seed,
+        if present
```

Comment:
kannst du hier noch in einem kleinen satz die eigenschaften des edi_seed_to_bo4e_mappings beschreiben? also
> key is a foo, value is a bar, example: ...

Reply:
Die Methode war von dir, ich hab die nur umgezogen 😜

Reply:
tja, wer hat das wohl reviewed? bie maus ist die chance groß, dass niemand :P

Reply:
b6ae16df58461f347f4aa5fce345029b56646281